test/bpf/verifier-test.sh

```diff
@@ -108,13 +107,8 @@ function cg_prog_type_init {
 
 	mkdir -p ${TESTMOUNT}
 
-	# These show up in sockops progs; skip them quietly.
-	prog_types["int"]="SKIP"
-	prog_types["_version"]="SKIP"
-	prog_types["version"]="SKIP"
 	if $BPFTOOL help 2>&1 | grep -q feature; then
 		if check_macro SK_MSG; then
-			prog_types["sk_msg"]="sock_ops"
 			attach_types["sk_msg"]="msg_verdict"
 		fi
```

Comment:
We can't removed just the `prog_types` entry here, this whole `if` block should be removed.